acm/registration/near-edge/base/near-edge.yaml

```diff
@@ -19,20 +19,15 @@ spec:
       - key: app
         operator: In
         values:
-          - near-edge
----
-apiVersion: v1
-kind: Namespace
-metadata:
-  name: ggithubcom-opendatahub-io-ai-edge-ns
+          - near-edge-acm-template
 ---
 apiVersion: apps.open-cluster-management.io/v1
 kind: Channel
 metadata:
   annotations:
     apps.open-cluster-management.io/reconcile-rate: medium
-  name: ggithubcom-opendatahub-io-ai-edge
```

Comment:
Can we rename from a non-generic `channel` change to show where it is pulling the Application from?

Reply:
@LaVLaS the generic `channel` name (and namespace) gets replaced by kustomize, so that it's an application-specific channel, in the same namespace as the subscription:
![image](https://github.com/opendatahub-io/ai-edge/assets/442386/c7be2726-7f89-46e9-bddf-dd5f29053bd1)

Currently those both point to the same place (this repo), but if I understood correctly yesterday, we want them to eventually point to Gitea, and they could be different repos there (this is what I was thinking when creating more than one channel here, rather than re-using the existing one that you had created manually for the odh-core application.

Would you like the structure or the names that this config defines to be different in some way?

Reply:
Thx for the clarification.  The use of `namePrefix` makes perfect since since we are using `kustomize`